clients/gen/common.sh

```diff
@@ -71,6 +71,8 @@ function run_pre_commit {
 
     # prepend license headers
     pre-commit run --all-files || true
+    echo "Output above will state Failed: 'Some sources were modified by the hook' - This is expected as the hook is used to add the licences."
+    pre-commit run --all-files
```

Comment:
@uranusjr any idea what signal a pre-commit failure would generate here?

Reply:
Since this function is only called from scripts without `set -e`, trap is not needed. But instead of `|| true` we should check `$?` and only show the message on failures.